parser.go

```diff
@@ -347,9 +347,8 @@ func (n *node) rebuild() {
 }
 
 func CheckExtension(path string, extensions []string) bool {
-	extension := filepath.Ext(path)
 	for _, item := range extensions {
-		if item == extension {
+		if path[len(path)-len(item):] == item {
```

Comment:
It will panic if `len(item) > len(path)`.

Reply:
You are right :)